api/views/__init__.py

```diff
@@ -0,0 +1,2 @@
+from . import storage, game_run
+__all__ = [storage, game_run]
```

Comment:
I think the elements of __all__ are supposed to be strings. (i.e. "storage", "game_run")